device_attributes/index.bs

```diff
@@ -0,0 +1,180 @@
+function successCallback(location) {
+  const tariff = backend.requestTariff(location);
+  console.log(tariff);
+}
+
+// Stop the workflow if the current environment is unexpected.
+function failureCallback(error) {
+  backend.reportFailure(error);
+  console.error(error.message);
+}
+
+function PrepareTariff() {
+  navigator.managed.getAnnotatedLocation()
+                   .then(successCallback, failureCallback);
+}
+</xmp>
+
+</div>
+
+It is easy to write another similar code snippet to report the sales data including a device serial number by using getAnnotatedAssetId() method. 
+
+NOTE: The website should assume that the data could be tampered with, stolen or reused on another managed device. Alternative security measures should be put in place by the website itself.)
+
+<div class="example">
+
+<xmp class="lang-js">
+function ReportSalesData() {
+  navigator.managed.getAnnotatedAssetId.then(reportCallback);
+}
+</xmp>
+
+</div>
+
+# Security considerations # {#security-considerations}
+
+In accordance with modern security practices, the permission of using these methods is controlled by the administrator per origin. In addition, they are only available to secure contexts on the managed devices.
+
+# Privacy considerations # {#privacy-considerations}
+
+This API provides access to device attributes which will be the same for all origins. (See <a href="https://wicg.github.io/WebApiDevice/managed_config/">MANAGED-CONFIG</a> for per-origin configuration of managed sites.) This could easily be used for tracking purposes if exposed arbitrarily. In a managed environment however, where an administrator is configuring a fleet of machines for particular employees or tasks, it is desirable for these machines to be able to identify themselves to the sites they are expected to be accessing.
+
+It is a non-goal for this interface to be used on unmanaged devices. An end user must never be asked to provide this information to a site. The decision of which origins are granted permission to access these device attributes must be made by an authorized administrator of the organization to which they belong. Most operating systems have a mechanism for the deployment of such policies, which can be used to configure many aspects of a device's operating system or applications such as the web browser. This specification requires that permission to access these device attributes is configured through such a system.
+
+To prevent unnecessary information exposure the configuration mechanism must require the administrator to grant access to these attributes to a limited set of origins, as granting access to all origins would create a mechanism for pervasive tracking of all web browsing on the device. Origins that are granted access must additionally be a <a href="https://html.spec.whatwg.org/multipage/webappapis.html#secure-context">secure context</a> so that passive network attackers are unable to observe these attributes in transit.
```

Comment:
Bikeshed knows how to link to the definition of "secure context".
```suggestion
To prevent unnecessary information exposure the configuration mechanism must require the administrator to grant access to these attributes to a limited set of origins, as granting access to all origins would create a mechanism for pervasive tracking of all web browsing on the device. Origins that are granted access must additionally be a [=secure context=] so that passive network attackers are unable to observe these attributes in transit.
```